README.md

```diff
@@ -1,93 +1,48 @@
-# Contributing
+# TypeScript Plugin for Autorest
 
-This project welcomes contributions and suggestions. Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.microsoft.com.
+[Autorest](https://github.com/Azure/autorest/blob/master/docs/readme.md) is a suite of tools to automatically generate SDKs for cloud services. This project provides a backend that generates SDKs in TypeScript.
 
-When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
+## Auto-generate your package in TypeScript using Autorest
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
-
-# Simple AutoRest Extension
+It is easy to generate an SDK once you have a swagger specification file.
 
-A simple AutoRest extension that can be used as a template for or documentation about writing such extensions.
-Since this a TypeScript project, we distinguish between language agnostic and language dependent requirements.
+- You will need first to install Autorest
 
-## Language Agnostic Requirements
-
-More information about these can be found [here](https://github.com/Azure/autorest/tree/master/docs/developer), summary:
-
-1. Specify the start command of the extension as the `start` script in the `package.json`
-2. Implement the AutoRest extension protocol (here: handled by library, see below)
-3. Hook up plugins into the AutoRest pipeline DAG, e.g.
+```bash
+npm install -g autorest

```

Comment:
Shouldnt this be `@autorest/typescript`?

Reply:
No, we use autorest itself and when you pass `--typescript` argument to it, it will use the TS extension.

Reply:
So, the typescript extension is installed automatically when we install `autorest`?

Reply:
it gets downloaded to a cache directory when you request it using the TS argument. You can also specify which version to use with the use arg: `--use=@microsoft.azure/autorest.typescript@4.4.4`